recipes-kernel/linux/linux-adi/0001-sc59x-fastautomdix-support.patch

```diff
@@ -0,0 +1,98 @@
+diff --git a/arch/arm/boot/dts/sc594-som-ezkit.dts b/arch/arm/boot/dts/sc594-som-ezkit.dts
+index 25acefc408c2..fd3d43c8eaa6 100644
+--- a/arch/arm/boot/dts/sc594-som-ezkit.dts
++++ b/arch/arm/boot/dts/sc594-som-ezkit.dts
+@@ -51,24 +51,22 @@
+ 			              <&ssw0 4 GPIO_ACTIVE_LOW>;	/* SPI2D2_D3_EN */
+ 		};
+ 
+-/* The button GPIO conflicts with OSPI0_D4 for this
++/* The button GPIO conflicts with OSPI0_D4 for this */
+ 		button0: button@0 {
```

Comment:
Button 0 and Button 1 both are enabled . Are both for push button ?

Reply:
Removed Button 1 section as it is not required